### PR TITLE
doc: sort ordering of lookup results

### PR DESCRIPTION
**Pull request by nickolay-kondratyev**
Should be merged after https://github.com/dendronhq/dendron/pull/2111

Related task note: [[dendron://private/task.lookup.2022.01.07.document-lookup-sort-order]]

### CODE REVIEW

File: vault/dendron.topic.lookup.find.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/lookup-cli.gif)
 
+## Sort Ordering
+When there are multiple matches they will be sorted in the following grouped order:
+  1. By FuseJS match score.
+  1. By whether the note stub or not (stubs go to the bottom of the score group).
```
Comment (jonathanyeung) [inline vault/dendron.topic.lookup.find.md:23]: nit: 'By whether the note _is a_ stub or not'

File: vault/dendron.topic.lookup.find.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
...
+  1. By levenshtein distance to original query (closer non-stub values go to the top within the score group).
```
Comment (jonathanyeung) [inline vault/dendron.topic.lookup.find.md:24]: nit: capitalize Levenshtein throughout

**Review by jonathanyeung — CHANGES_REQUESTED** [review]

File: vault/dendron.topic.lookup.find.md (hunk 1 above)
Comment (nickolay-kondratyev) [reply vault/dendron.topic.lookup.find.md:23]: If we know a good spell checker extension, we should add a suggestion in our https://wiki.dendron.so/notes/301e4129-6933-4be7-a4bd-8125171360d8/ extension page (If that is the page where we expect to find extra extensions to add into VS Code)

File: vault/dendron.topic.lookup.find.md (hunk 2 above)
Comment (nickolay-kondratyev) [reply vault/dendron.topic.lookup.find.md]: Addressed

**Comment by jonathanyeung** [comment]
lgtm, thanks!

File: vault/dendron.topic.lookup.find.md [hunk 3]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/lookup-cli.gif)
 
+## Sort Ordering
+When there are multiple matches they will be sorted in the following grouped order:
+  1. By FuseJS match score.
+  1. By whether the note is a stub or not (stubs go to the bottom of the score group).
+  1. By Levenshtein distance to original query (closer non-stub values go to the top within the score group).
```
Comment (ScriptAutomate) [inline vault/dendron.topic.lookup.find.md:24]:
```suggestion
  1. By [Levenshtein distance](https://medium.com/@ethannam/understanding-the-levenshtein-distance-equation-for-beginners-c4285a5604f0) to original query (closer non-stub values go to the top within the score group).
```

File: vault/dendron.topic.lookup.find.md [hunk 4]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/lookup-cli.gif)
 
+## Sort Ordering
+When there are multiple matches they will be sorted in the following grouped order:
+  1. By FuseJS match score.
```
Comment (ScriptAutomate) [inline vault/dendron.topic.lookup.find.md:22]:
```suggestion
  1. By [FuseJS match score](https://fusejs.io/concepts/scoring-theory.html).
```

File: vault/dendron.topic.lookup.find.md [hunk 5]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
...
+  1. By whether the note is a stub or not (stubs go to the bottom of the score group).
```
Comment (ScriptAutomate) [inline vault/dendron.topic.lookup.find.md:23]:
```suggestion
  1. By whether the note is a [[stub|dendron://dendron.dendron-site/dendron.concepts#stubs]] or not (stubs go to the bottom of the score group).
```

File: vault/dendron.topic.lookup.find.md [hunk 6]
```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
...
+  1. By Levenshtein distance to original query (closer non-stub values go to the top within the score group).
+  1. By last update time. (Most recently updated notes with same Levenshtein distance will bubble up within score group).
```
Comment (ScriptAutomate) [inline vault/dendron.topic.lookup.find.md:25]:
```suggestion
  1. By last [[update time|dendron://dendron.dendron-site/dendron.topic.frontmatter#updated]]. (Most recently updated notes with same Levenshtein distance will bubble up within score group). 
```

**Review by ScriptAutomate — APPROVED** [review]

**Pull request merged**